fixmydjango/urls.py

```diff
@@ -31,6 +34,23 @@ def activate(request, uid, token):
 
 
 urlpatterns = [
+    path(
+        'openapi',
+        get_schema_view(
+            title="FMC API",
```

Comment:
May you can investigate if we can set this based on the app configuration (`REGION` env variable)? Since this repo serves three different applications whose APIs are mentally, but not logical separated.